lms/static/certificates/sass/_components.scss

```diff
@@ -556,7 +556,7 @@
     top: 0;
     left: 0;
     z-index: z-index(front);
-    background-color: palette(grayscale, trans); /* dim the background */
+    background-color: transparentize(palette(grayscale, base), 0.5); /* dim the background */
```

Comment:
Again, my understanding is that this is making the gray transparent, not lightening it.